### ISSUE TITLE
Lando not pulling Pantheon files

### ISSUE THREAD

**Issue by adendrag** [lando/lando#1330]
Bug Report
==========

**Tell us about your setup**

Lando v3.0.0-beta.47 on OSX 10.14.1

```
name: eg-test-aden
recipe: pantheon
config:
  framework: drupal8
  site: eg-test-aden
  id: 78d59530-b580-4e26-88c2-fb5e90869e17
```

**Tell us about the command you were running**

```
lando pull
```

**Tell us about the error you got**

```
Pulling files from dev...
receiving incremental file list
rsync: mkdir "/app/sites/default/files" failed: No such file or directory (2)
rsync error: error in file IO (code 11) at main.c(674) [Receiver=3.1.1]

Pull complete!
verbose: Emitting event post-engine-run
debug: Event post-engine-run has 0 listeners
verbose: Emitting event post-pull
debug: Event post-pull has 0 listeners
```

Created a new Pantheon site and tried to run lando pull but it does not pull the site files.

**Comment by danielrose28** [comment]
I'm seeing this error as well with a site I started setting up locally with lando last week. Haven't figured it out yet though. :(

**Comment by ccharlton** [comment]
Are you sure the [Lando app] path is correct? I see `/app/sites/default/files` but I assume for D8 it would be `/app/web/sites/default/files` (missing the `web`) portion of the path. But I don't see that in your config; it's common but not required.

**Comment by pirog** [comment]
This should be fixed as a consequence of #1325
